core/src/main/java/com/amplifyframework/core/model/AWSTimestamp.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core.model;
+
+/**
+ * The AWSTimestamp scalar type represents the number of seconds that have elapsed
+ * since 1970-01-01T00:00Z. Timestamps are serialized and deserialized as numbers.
+ * Negative values are also accepted and these represent the number of seconds
+ * til 1970-01-01T00:00Z.
+ */
+public final class AWSTimestamp {
```

Comment:
I think it would be helpful to provide additional constructors like 
```
public AWSTimestamp(Date date);
public AWSTimestamp(String text);
```

I think we should try to strongly associate `Date` with our new formats since it is what the customers are likely more familiar with. Our other AWS temporal datatypes are already coupled with `Date` if I remember correctly.

Reply:
Good idea, will do

Reply:
I ended up with:
```
// "now"
AWSTimestamp timestamp = new AWSTimestamp();

// Any date
AWSTimestamp timestamp = new AWSTimestamp(new Date());

// An amount of time since the epoch
AWSTimestamp timestamp = new AWSTimestamp(5_000L, TimeUnit.MILLISECONDS);
```